luda/editor/client/src/index.ts

```diff
@@ -25,6 +26,42 @@ startEngine(
       tracks: [
         {
           id: "track1",
+          type: TrackType.subtitle,
+          clips: [
+            {
```

Comment:
I recommend using `createClip` for this if you not test something with custom data.

Reply:
Fixed
with https://github.com/NamseEnt/namseent/pull/18/commits/27f61849cfa8272b4969baa72cee0d263caa4e7a